src/i18n/en.json

```diff
@@ -185,7 +185,7 @@
     },
 
     "footer": {
-      "copyright": "2018 © Distributed Lab.",
+      "copyright": "2019 © Distributed Lab.",
```

Comment:
dynamically calculate year in the component?

Reply:
added `currentYear` computed poperty